docs/installation/install-git-clone.md

```diff
@@ -43,9 +59,24 @@ git checkout master
 php composer.phar install
 ```
 
+
 Next Steps
 ----------
 
+### Example Themes
+
+When a `composer install` or `composer update` is run, **without** a
+`composer.lock` present in the site root, Bolt will install the example themes
+from `bolt/themes`.
+
+<p class="note"><strong>Note:</strong> These themes are required to run both
+unit & acceptance tests.</p>
+
+Should you require updated copies of these themes, they can be resynchronised
+with the copies in the vendor directory via the Nut [`setup:sync`][nut-setup-sync]
```

Comment:
Well, that was a consideration initially … but we're trying to move away from bits scattered around and point people to The One True Answer :wink: … moreover keeping one place to maintain.

I'm not "holding firm" on that, just trying to stick with current agreements with the team.